docs/browser.md

```diff
@@ -7,7 +7,7 @@ Run Prettier in the browser using its **standalone** version. This version doesn
 
 The standalone version comes as:
 
-- ES modules: `esm/standalone.mjs`, starting in version 2.2
+- ES modules: `standalone.mjs`, starting in version 2.2
```

Comment:
Since `standalone.mjs` does not exist in version 2.x, it would be good to clarify this.

```suggestion
- ES modules: `standalone.mjs`, starting in version 3.0 (In version 2, `esm/standalone.mjs`.)
```